src/rpc/cli.cpp

```diff
@@ -14,8 +14,16 @@
 # endif
 #endif
 
+#include <boost/regex.hpp>
+
 namespace fc { namespace rpc {
 
+static std::string& cli_regex_secret()
+{
+   static std::string* regex_secret = new std::string();
```

Comment:
No need to use ``new`` operator here. Just let regex_secret be a ``std::string`` and return that.

Reply:
ok ! Thanks !